src/Management/src/Endpoint/Actuators/Health/HealthEndpointOptions.cs

```diff
@@ -8,9 +8,29 @@ namespace Steeltoe.Management.Endpoint.Actuators.Health;
 
 public sealed class HealthEndpointOptions : EndpointOptions
 {
-    public ShowDetails ShowDetails { get; set; }
+    /// <summary>
+    /// Gets or sets whether to show components in responses.
+    /// </summary>
+    public ShowValues ShowComponents { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether to show details of components in responses.
+    /// </summary>
+    public ShowValues ShowDetails { get; set; }
+
+    /// <summary>
+    /// Gets or sets the claim requirements for using this endpoint.
+    /// </summary>
     public EndpointClaim? Claim { get; set; }
+
+    /// <summary>
+    /// Gets or sets the role required to use this endpoint.
```

Comment:
```suggestion
    /// Gets or sets the role required to retrieve components and/or details.
```
From what I understand, this setting doesn't affect whether the endpoint can be used.